its/plugin/src/test/java/com/sonar/python/it/plugin/MetricsTest.java

```diff
@@ -55,85 +81,88 @@ public static void startServer() {
   @Test
   public void project_level() {
     // Size
-    assertThat(getProjectMeasureAsInt("ncloc")).isEqualTo(1);
-    assertThat(getProjectMeasureAsInt("lines")).isEqualTo(6);
-    assertThat(getProjectMeasureAsInt("files")).isEqualTo(1);
-    assertThat(getProjectMeasureAsInt("statements")).isEqualTo(1);
-    assertThat(getProjectMeasureAsInt("functions")).isEqualTo(0);
-    assertThat(getProjectMeasureAsInt("classes")).isEqualTo(0);
+    assertThat(getProjectMeasureAsInt(NCLOC)).isEqualTo(1);
+    assertThat(getProjectMeasureAsInt(LINES)).isEqualTo(6);
+    assertThat(getProjectMeasureAsInt(FILES)).isEqualTo(1);
+    assertThat(getProjectMeasureAsInt(STATEMENTS)).isEqualTo(1);
+    assertThat(getProjectMeasureAsInt(FUNCTIONS)).isEqualTo(1);
+    assertThat(getProjectMeasureAsInt(CLASSES)).isEqualTo(0);
     // Documentation
-    assertThat(getProjectMeasureAsInt("comment_lines")).isEqualTo(1);
-    assertThat(getProjectMeasureAsDouble("comment_lines_density")).isEqualTo(50.0);
+    assertThat(getProjectMeasureAsInt(COMMENT_LINES)).isEqualTo(1);
+    assertThat(getProjectMeasureAsDouble(COMMENT_LINES_DENSITY)).isEqualTo(50.0);
     // Complexity
-    assertThat(getProjectMeasureAsDouble("complexity")).isZero();
-    assertThat(getProjectMeasureAsDouble("function_complexity")).isNull();
-    assertThat(getProjectMeasure("function_complexity_distribution").getValue()).isEqualTo("1=0;2=0;4=0;6=0;8=0;10=0;12=0;20=0;30=0");
-    assertThat(getProjectMeasureAsDouble("file_complexity")).isZero();
-    assertThat(getProjectMeasure("file_complexity_distribution").getValue()).isEqualTo("0=1;5=0;10=0;20=0;30=0;60=0;90=0");
+    assertThat(getProjectMeasureAsDouble(COMPLEXITY)).isEqualTo(2.0);
+    assertThat(getProjectMeasureAsDouble(COGNITIVE_COMPLEXITY)).isEqualTo(1.0);
```

Comment:
I would have liked to see another file to get another value for cognitive complexity here, showing that it is now indeed computed on whole file/project and not only at functions level anymore

Reply:
done